wrappers/python/xraydb/xraydb.py

```diff
@@ -258,16 +281,18 @@ def f0(self, ion, q):
     def _getChantler(self, element, energy, column='f1', smoothing=1):
         """
         return energy-dependent data from Chantler table
+
+        Parameters:
+            element (string or int): atomic number or symbol.
+            eneregy (float or ndarray):
         columns: f1, f2, mu_photo, mu_incoh, mu_total
 
-        Note: this function is meant for internal use.
+        Notes:
+           this function is meant for internal use.
```

Comment:
You can put note this way:
```
.. note:: Deprecated in Numpy 1.6
          `ndobj_old` will be removed in Numpy 2.0, it is replaced by
          `ndobj_new` because the latter works also with array subclasses.
```

from https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt